doc/v2.md

```diff
@@ -0,0 +1,20 @@
+## Guide for Migration from v1 to v2
+
+`v2` contains lots of refactoral changes to improve the performance and configuration by introducing [`@logdna/logger`](https://www.npmjs.com/package/@logdna/logger), [`@logdna/env-config`](https://www.npmjs.com/package/@logdna/env-config), and [`eslint-config-logdna`](https://www.npmjs.com/package/eslint-config-logdna). There are several changes in naming and handling environment variables as well. All environment variables are documented [here](./env.md)
+
+### Renamed Variables
+* `LOGDNA_KEY` has been replaced with [`INGESTION_KEY`](./env.md#ingestion_key)
+* `LOGDNA_HOSTNAME` has been replaced with [`HOSTNAME`](./env.md#hostname)
```

Comment:
Not absolutely necessary, but would be better, replace "has been replaced" with "was replaced" throughout this section.

Reply:
Done!